audioplayer.h

```diff
@@ -0,0 +1,44 @@
+#include <Windows.h>
+#include <MMSystem.h>
+#include <iostream>
+#include <io.h>
+#include <dirent.h>
+#include <vector>
+#include <string>
+#include <algorithm>
+
+#pragma once
+#pragma comment(lib, "winMM.lib")
+
+using namespace std;
+
+void callMciSendString(const string &str);
+
+vector<string> getFileNames(const string &);
+
+class AudioPlayer {
+
+private:
+    string trackName;
+    string fileLink;
+    string currentVolume = "50";
+    bool isPlaying = false;
+
+public:
+
+    void playAudio();
+
+    void changeVolume(string);
+
+    void rewindAudio(string);
+
+    void closeAudio();
+
+    bool changeAudio(int);
```

Comment:
You need variable names in here too. Replace it to `int trackNumber`. In other methods too.

Reply:
done